src/Target.php

```diff
@@ -234,7 +240,7 @@ private function substitute($name, Message $message)
             if ($config['short']) {
                 $value = '`' . $value . '`';
             } else {
-                $value = "```text\n" . $value . "\n```";
+                $value = "```text\n" . StringHelper::truncate($value, $this->messageMaxLength) . "\n```";
```

Comment:
It'd be safer to truncate a value anyway, whether it's declared as short or not.